### DISCUSSION TITLE
How can I create login page

### DISCUSSION THREAD

**Discussion by cfsdes** [Budibase/budibase#1999] [Help] [answered]
Hi,

I searched the discussion threads but I couldn't find it. I understand that users are managed in budbase itself, but how do I create a login page for these users to login?

**Comment by shogunpurple** [comment, marked as answer]
Hi @cfsdes -

This one is answered in https://github.com/Budibase/budibase/discussions/1975.

In a nutshell, budibase is built for internal tools within the context of an organisation, rather than external customer facing SaaS applications or products. For this reason a lot of our authentication is managed by budibase and remains opinionated. In order to invite users, the admin of your budibase installation must specifically invite them to the organisation.

**Reply by fatherofinvention** [reply]
Hey @shogunpurple 

Is it possible for this to ever be reconsidered? Budibase is an excellent tool that I would gladly pay for if it would allow me to use a traditional user sign-up and authentication flow as you described in your external facing SaaS example. I understand if this is not your focus today, but hopefully you and the team will revisit it as a possibility sometime in the future. Keep up the good work.

**Reply by shogunpurple** [reply]
Hi @fatherofinvention - thanks for the kind words. Unfortunately, a platform for building SaaS tools is not in the direction or vision of the company right now. As time goes on, we are optimising for tools to optimise workflows in businesses and to allow companies to build internal applications faster than ever before.

We will also be implementing per user pricing for the paid tiers, which obviously is not a good fit for a SaaS application with a public sign up page. 

Thank you for raising this!
